core/designsystem/src/main/java/org/sopt/designsystem/component/dialog/SoptAlertDialog.kt

```diff
@@ -0,0 +1,37 @@
+package org.sopt.designsystem.component.dialog
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+
+@Composable
+fun SoptAlertDialog(
+    title: String,
+    positiveLabel: String,
+    negativeLabel: String,
+    onClickPositive: () -> Unit,
+    onClickNegative: () -> Unit,
+){
+    AlertDialog(
+        onDismissRequest = { onClickNegative() },
```

Comment:
```suggestion
        onDismissRequest = onClickNegative,
```

취향차이인데 이렇게 쓸 수도 있습니다.